components/director/internal/model/eventapi.go

```diff
@@ -96,6 +97,7 @@ type EventDefinitionInput struct {
 	DefaultConsumptionBundle *string                       `json:"defaultConsumptionBundle"`
 	Hierarchy                json.RawMessage               `json:"hierarchy"`
 	DocumentationLabels      json.RawMessage               `json:"documentationLabels"`
+	CorrelationIDs           json.RawMessage               `json:"correlationIds,omitempty"`
```

Comment:
Make also "resourceDefinitions" optional.
"Made resourceDefinitions optional for Event Resources, to be consistent with API Resources"